src/pkg/message/progress.go

```diff
@@ -40,6 +40,20 @@ func NewProgressBar(total int64, text string) *ProgressBar {
 	}
 }
 
+// Close doesn't do anything but satisfy implementation.
+// https://github.com/defenseunicorns/pkg/blob/325e69d2560e8d767511c9e27187f775ecbad3fa/helpers/progress.go#L31-L38
+func (*ProgressBar) Close() error {
+	return nil
+}
+
+// Updatef doesn't do anything but satisfy implementation.
+// https://github.com/defenseunicorns/pkg/blob/325e69d2560e8d767511c9e27187f775ecbad3fa/helpers/progress.go#L31-L38
+func (*ProgressBar) Updatef(_ string, _ ...any) {}
```

Comment:
We do want Updatef to do things. We should change the update function to updatef

Reply:
Actually we should keep the Update function, but updatef should be implemented.

Reply:
implemented Updatef